<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShoppingManager</class>
 <widget class="QWidget" name="ShoppingManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>潮海波の店铺管理</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background: #8360c3;  /* fallback for old browsers */
background:qlineargradient(spread:pad,x1:1,y1:0,x2:0,y2:0,stop:0 #8360c3,stop:1 #2ebf91);
color:rgb(255, 255, 255);
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background:transparent;
font: 20pt &quot;黑体&quot;;</string>
        </property>
        <property name="text">
         <string> 店铺管理系统</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Ubuntu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AMOLED</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Aqua</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ElegantDark</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MacOS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ManjaroMix</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MaterialDark</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NeonButtons</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="SlideNavigation" name="SideNav" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QStackedWidget" name="PageStack">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="ProductPage" name="ProductP"/>
     <widget class="OrderPage" name="OrderP"/>
     <widget class="ContactPage" name="ContactP"/>
     <widget class="ProsessPage" name="ProsessP"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SlideNavigation</class>
   <extends>QWidget</extends>
   <header>slidenavigation.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ContactPage</class>
   <extends>QWidget</extends>
   <header>contactpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProductPage</class>
   <extends>QWidget</extends>
   <header>productpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OrderPage</class>
   <extends>QWidget</extends>
   <header>orderpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProsessPage</class>
   <extends>QWidget</extends>
   <header>prosesspage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
